src/components/Filters/ButtonGroup.tsx

```diff
@@ -8,12 +8,14 @@ type ButtonGroupProps = {
   options: string[];
   selectedKeys: string[];
   toggleDimension: (dimension: string) => void;
+  displayOptions?: { [key: string]: string };
```

Comment:
This is a good solution. It allows us to be flexible and change the button text without worrying about changing the button's value.